usaspending_api/api_contracts/contracts/v2/references/award_types.md

```diff
@@ -0,0 +1,89 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Award Types [/api/v2/references/award_types/]
+
+This endpoint returns a JSON object representing the grouping of award types.
+​
+## GET    
++ Response 200 (application/json)
+    + Attributes (object)
+        + `contracts` (required, ContractTypes, fixed-type)
+        + `loans` (required, LoanTypes, fixed-type)
+        + `idvs` (required, IDVTypes, fixed-type)
+        + `grants` (required, GrantTypes, fixed-type)
+        + `other_financial_assistance` (required, OtherFinancialAssistanceTypes, fixed-type)
+        + `direct_payments` (required, DirectPaymentTypes, fixed-type)
+    + Body
+
+                {
```

Comment:
Can you correct the formatting of this body? Screenshot attached showing what it currently looks like: 
![Screen Shot 2020-04-17 at 8 03 18 AM](https://user-images.githubusercontent.com/40359517/79583692-416d3000-8082-11ea-9343-87c9c9e93956.png)

Reply:
K